src/dialog.tsx

```diff
@@ -110,19 +110,17 @@ const DialogContent = (props: any): JSX.Element => {
             method: 'POST'
         }).then(reply => {
             var result = reply["deployment_result"]
-
-            if (result.message) {
-                setDeployErrorMessage(result.message)
+            if (result.detail || result.message) {
+                var errorMsg = result.detail || result.message
+                setDeployErrorMessage(errorMsg)
```

Comment:
@edublancas 
The frontend is consuming two type of error messages from backend Endpoint:
That's why I need to consolidate `detail` or `message` here

A. API Key is invalid -
```
{
    "detail": "Could not validate API KEY"
}
```

B. Only allow one active project
```
{
    "message": "Community users are only allowed to have a single active project. Delete your current project to create a new one."
}
```

Is possible they can consolidate in the backend?

Reply:
thanks for letting me know. I'll open an issue